src/bci_build/package/__init__.py

```diff
@@ -1385,6 +1309,23 @@ def __post_init__(self) -> None:
         if not self.version:
             raise ValueError("A language stack container requires a version")
 
+    @property
+    def distribution_base_name(self) -> str:
+        if self.os_version.is_tumbleweed:
+            return super().distribution_base_name
+        return "SLE"
+
+    @property
+    def eula(self) -> str:
+        """EULA covering this image. Development containers are always ``sle-bci``."""
+        return "sle-bci"
```

Comment:
As we have no LTSS development containers, this shouldn't need an override

Reply:
already explained earlier. the unit test suite is testing DevelopmentContainers for LTSS, which does not happen in practice, but if it happens then this change is needed. 

alternatively review #1399 please